info/v1/container.go

```diff
@@ -110,6 +110,10 @@ type ContainerInfoRequest struct {
 	// End time for which to query information.
 	// If ommitted, current time is assumed.
 	End time.Time `json:"end,omitempty"`
+
+	// Flag to determine if expanded subcontainer stats are requested.
+	// If ommitted, assume false.
+	ExpandSubcontainers bool `json:"expand_subcontainers,omitempty"`
```

Comment:
There's a separate endpoint for this: `http://cadvisor:8080/api/v1.1/subcontainers` (note that it's not available in 1.0).